stable/localvolumeprovisioner/values.yaml

```diff
@@ -3,10 +3,11 @@ image:
   tag: "v2.3.2"
   pullPolicy: "IfNotPresent"
 
-storageclass:
-  isDefault: true
-  reclaimPolicy: Delete
-  volumeBindingMode: WaitForFirstConsumer
+storageclasses:
```

Comment:
I feel like this would be a great place to add some comments documenting how this configuration can be used and what type of problems it's meant to solve (e.g. the ability to have more than one storageclass deployed by localvolumeprovisioner).

Reply:
Good point! I'll add a comment here.